services/crypto/key/threats.yaml

```
@@ -1,19 +1,26 @@
-common_threats:
-  - CCC.TH01 # Access Control is Misconfigured
-  - CCC.TH04 # Data is Replicated to Untrusted or External Locations
+shared-threats:
+  - reference-id: CCC
+    identifiers:
+      - CCC.TH01 # Access Control is Misconfigured
+      - CCC.TH04 # Data is Replicated to Untrusted or External Locations
+
 threats:
-  - id: CCC.KeyMgmt.TH01
+  - id: CCC.KeyMgmt.TH17
```

Can you review this change? I think I changed this from `TH17` to `TH01` as there was only one threat - and was possibly reverted in a merge?

---

```suggestion
  - id: CCC.KeyMgmt.TH01
```

I'm not finding the cause, and I can only hope that this was a one-off 😬